lib/cs_guide/resources/venue.ex

```diff
@@ -9,11 +9,19 @@ defmodule CsGuide.Resources.Venue do
 
   schema "venues" do
     field(:venue_name, :string)
-    field(:postcode, :string)
-    field(:phone_number, :string)
+    field(:postcode, Fields.Postcode)
+    field(:phone_number, Fields.PhoneNumber)
     field(:cs_score, :float, default: 0.0)
+    field(:num_cocktails, :integer)
     field(:entry_id, :string)
     field(:deleted, :boolean, default: false)
+    field(:description, :string)
+    field(:website, :string)
+    field(:address_line_1, Fields.Address)
```

Comment:
this field does not need to be called `:address_line_1` because we only _allow_ one field of address information which can have as many lines as the user wants. `Fields.Address` should be rendered as a `<textarea>` with one row which the user can choose to expand if their address is multi-line.